src/main/java/com/pirimid/cryptotrade/helper/exchange/gemini/dto/response/CreateOrderResponse.java

```diff
@@ -0,0 +1,35 @@
+package com.pirimid.cryptotrade.helper.exchange.gemini.dto.response;
```

Comment:
this DTO is for getting response from exchange.right?

Reply:
Yes